src/styles/calculator.css

```diff
@@ -0,0 +1,106 @@
+/* TODO: Convert to SCSS */
```

Comment:
You can just rename it to a .scss file without any changes if you want. Scss is a superset of CSS

Reply:
oh wow, today i learned something haha